code/upaya/app/assets/javascripts/idp_help.js.erb

```diff
@@ -0,0 +1,16 @@
+$(document).on('page:change', function () {
+  var slideTime = 300;
+
+  $('#help-button').click(function () {
+    if ($('#help-content').hasClass('open')) {
+      $('#help-content').removeClass('open');
+      $('#help-content').slideUp(slideTime);
+      $('#help-label').html("<%= raw(I18n.t('idp.labels.where_find') + I18n.t('idp.labels.where_find_hidden')) %>");
+    } else {
+      $('#help-content').addClass('open');
+      $('#help-content').slideDown(slideTime);
+      $('#help-label').html("<%= raw(I18n.t('idp.labels.hide_finder') + I18n.t('idp.labels.hide_finder_hidden')) %>");
+      $('#help-content').removeClass('slide-panel');
+    }
+  });
+});
```

Comment:
I agree with you that it's best to separate, I was just doing forklift scale moves from the other repo to get us a base to work from. It's fine if we want to leave those bits out of this PR and add in a followup since this is becoming unwieldy (the PR takes 1-2s to load in the browser)